tests/review-child-handoff.test.sh

```
@@ -60,6 +442,61 @@ simplify_context_out="$(uberdev_agent_context_create "$TMP/simplify-run" "$simpl
+# constructor rejects it before publishing a child directory.
+MAX_PLUS_ONE_INPUTS="$(python3 -I -B - "$MAX_INPUTS" "$TREE" <<'PY'
+import json,sys
+value=json.loads(sys.argv[1])
+limit=json.load(open(sys.argv[2]))['input_limits']['max_serialized_bytes']
+value["changed_paths"][-1]=value["changed_paths"][-1][:-3]+"x.ts"
+payload=json.dumps(value,sort_keys=True,separators=(",",":"))
+assert len(payload.encode())==limit+1
+print(payload,end="")
+PY
+)"
+! uberdev_create_child_handoff review_pr.review.correctness review-constructor-plus-one-iter1-attempt01 "$MAX_PLUS_ONE_INPUTS" '[]' >/dev/null 2>&1
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)